src/media/css/newsletter.styl

```diff
@@ -204,9 +204,9 @@ $arrow-size = 10px;
     }
     .newsletter-signup-button,
     .newsletter-cancel-button {
-        width: auto;
-        margin: 0;
         border-radius: 5px;
+        margin: 0;
+        width: auto;
```

Comment:
You could have avoided adding these changes from diffs I think, but it should be fine for now.